app/components/map.js

```diff
@@ -192,7 +167,7 @@ function getEventsMapContent(locations) {
                 longitude: maxDistanceTeamLocation.longitude,
             }}/>;
     });
-    const teamPolylines = locations.map(team => {
+    const teamPolylines = props.locations.map(team => {
```

Comment:
do we need these since they're basically the event starting point?

Reply:
these are all the polylines foreach team